packages/react/src/components/InAppMessaging/MessageLayout/MessageLayout.tsx

```diff
@@ -89,7 +90,14 @@ export function MessageLayout({
           data-testid={TEXT_CONTAINER_TEST_ID}
         >
           {header?.content && (
-            <Heading style={styles.header}>{header.content}</Heading>
+            <Heading
+              className={headerClass}
+              isTruncated
```

Comment:
Need to merge in `main` to pull in [this change](https://github.com/aws-amplify/amplify-ui/pull/2371)